src/coreclr/src/tools/crossgen2/ILCompiler.ReadyToRun/Compiler/DependencyAnalysis/ReadyToRun/DelegateCtorSignature.cs

```diff
@@ -89,7 +89,7 @@ public override int CompareToImpl(ISortableNode other, CompilerComparer comparer
             if (result != 0)
                 return result;
 
-            result = comparer.Compare(_targetMethod.Method, otherNode._targetMethod.Method);
+            result = _targetMethod.CompareToImpl(otherNode._targetMethod, comparer);
```

Comment:
Wouldn't this work? CompareToImpl is kind of an implementation detail. Calling `Compare` has a couple safeties in it.

```suggestion
            result = comparer.Compare(_targetMethod, otherNode._targetMethod);
```

Reply:
Fixed in 3rd commit, thanks for pointing that out.